projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -46,22 +47,27 @@ export class CartesianWidgetRendererComponent<TData> extends InteractiveDataWidg
 
   public selectedInterval?: IntervalValue;
   public intervalOptions?: IntervalValue[];
-  private fetcher?: MetricSeriesFetcher<TData>;
+  private seriesFetcher?: MetricSeriesFetcher<TData>;
```

Comment:
I'm going to defer reviewing the dashboard code now, since that's really dependent on the graphql api. Can we split that into a separate PR?